index.bs

```diff
@@ -2387,6 +2387,30 @@ Note: Invoking this method from a [=browsing context=] where the [=Web Authentic
 
 </div>
 
+### Deserialize Registration and Authentication ceremony options - PublicKeyCredential's `optionsFromJSON()` Method ### {#sctn-optionsFromJSON}
```

Comment:
This section should include an IDL definition like in the `isUserVerifyingPlatformAuthenticatorAvailable()` section, and declare the parameter like the other sibling sections do.

Reply:
Absolutely! I've been working with @dwaite behind the scenes to prepare IDL definitions for these new methods; now that a PR exists we can start that work in earnest.